### PR TITLE
used env var for new owner

### PR DESCRIPTION
**Pull request by rahul-ramesh**

### CODE REVIEW

**Comment by rahul-ramesh** [comment]
Pull the new owner ID from the env if present, otherwise use default. Tested locally. First js foray so appreciate any feedback :)

**Comment by glitch003** [comment]
This looks great!  Can you also apply the same changes to "deploy_everything.js".  I think we will switch back to that, away from the solonet stuff, soon.  

alternatively, you can see what's different between "deploy_everything.js" and "deploy_everything_for_solonet.js" (which should be minimal) and combine them into a single file.  Maybe combine them into "deploy_everything.js" and then put in a parameter to switch regular and solonet.

**Comment by rahul-ramesh** [comment]
> This looks great! Can you also apply the same changes to "deploy_everything.js". I think we will switch back to that, away from the solonet stuff, soon.
> 
> alternatively, you can see what's different between "deploy_everything.js" and "deploy_everything_for_solonet.js" (which should be minimal) and combine them into a single file. Maybe combine them into "deploy_everything.js" and then put in a parameter to switch regular and solonet.

Updated!

**Comment by rahul-ramesh** [comment]
@glitch003 should I delete the solonet version, or should we keep it around for posterity? (I vote for option 2 until we fully deprecate)

File: scripts/deploy_everything.js [hunk 1]
```diff
@@ -108,6 +104,9 @@ async function main() {
         process.exit(1);
     }
 
+    var solonet = process.env.LIT_SOLONET || true;
```
[the remark this answers is not in the document]
Comment (rahul-ramesh) [reply scripts/deploy_everything.js]: I've updated the notion readme

**Comment by glitch003** [comment]
Yeah let's keep it around in this PR and I'll test everything and if it's good to go then i'll delete it

**Pull request merged**